packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -59,37 +118,55 @@ QUnit.module('init', {
         assert.equal($content.find(MESSAGE_SELECTOR).text(), 'Test Loading Message');
     });
 
-    QUnit.test('role on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.test('correct wrapper aria attributes load when showIndicator = false', function(assert) {
+        const label = messageLocalization.format('Loading');
+        const loadPanel = $('#loadPanel').dxLoadPanel({
+            visible: true,
+            showIndicator: false
+        }).dxLoadPanel('instance');
+
+        const $wrapper = loadPanel.$wrapper();
 
-        assert.strictEqual(instance.$wrapper().attr('role'), 'alert');
+        assert.strictEqual($wrapper.attr('role'), 'alert', 'role attribute has value alert ');
+        assert.strictEqual($wrapper.attr('aria-label'), label, 'aria-label attribute has value Loading...');
     });
 
-    QUnit.test('aria-label on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.test('wrapper aria attributes must not load when showIndicator = true', function(assert) {
+        const $loadPanel = $('#loadPanel').dxLoadPanel({
+            visible: true,
+            showIndicator: true
+        });
+
+        assert.strictEqual($loadPanel.attr('role'), undefined, 'role is not set');
+        assert.strictEqual($loadPanel.attr('aria-label'), undefined, 'aria-label is not set');
+    });
+
+    QUnit.test('correct load indicator aria attributes load when showIndicator = true', function(assert) {
+        const loadPanel = $('#loadPanel').dxLoadPanel({
+            visible: true,
+            showIndicator: true
+        }).dxLoadPanel('instance');
 
-        assert.strictEqual(instance.$wrapper().attr('aria-label'), undefined);
+        const label = messageLocalization.format('Loading');
 
-        instance.option({ message: '' });
+        const $indicator = loadPanel.$content().find(LOADINDICATOR_CLASS);
 
-        assert.strictEqual(instance.$wrapper().attr('aria-label'), 'Loading...');
+        assert.strictEqual($indicator.attr('role'), 'alert', 'role attribute has value alert');
+        assert.strictEqual($indicator.attr('aria-label'), label, 'aria-label attribute has value Loading...');
     });
 
-    QUnit.test('aria-label on wrapper in Material based themes', function(assert) {
-        const origIsMaterial = themes.isMaterialBased;
-        themes.isMaterialBased = () => true;
+    QUnit.test('aria-label on wrapper', function(assert) {
```

Comment:
Seems like this test is not necessary anymore because you check it in "correct wrapper aria attributes load when showIndicator = false" test. How it can be red if the mentions test is green?

Reply:
[7c89496](https://github.com/DevExpress/DevExtreme/pull/27080/commits/7c89496fe5ea9a3a695c7dcce3717057deeca65c)